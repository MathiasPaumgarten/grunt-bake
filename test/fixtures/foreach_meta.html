<dl>
	<!--(bake-start _foreach="name:[robert, susan, carl]")-->
	<dt>{{name}}</dt>
	<dd>
		Index: {{name@index}}<br>
		Iteration: {{name@iteration}}<br>
		First: {{name@first}}<br>
		Last: {{name@last}}<br>
		Total: {{name@total}}
	</dd>
	<!--(bake-end)-->
</dl>