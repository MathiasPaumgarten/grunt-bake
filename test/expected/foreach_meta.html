<dl>
	<dt>robert</dt>
	<dd>
		Index: 0<br>
		Iteration: 1<br>
		First: true<br>
		Last: false<br>
		Total: 3
	</dd>
	<dt>susan</dt>
	<dd>
		Index: 1<br>
		Iteration: 2<br>
		First: false<br>
		Last: false<br>
		Total: 3
	</dd>
	<dt>carl</dt>
	<dd>
		Index: 2<br>
		Iteration: 3<br>
		First: false<br>
		Last: true<br>
		Total: 3
	</dd>
</dl>